### ISSUE TITLE
Add warning when installing custom container

### ISSUE THREAD

**Issue by kkauder** [eic/eic-shell#7]
**Is your feature request related to a problem? Please describe.**
On linux, `singularity pull` creates a multi-GB cache in ~/.singularity

**Describe the solution you'd like**
That's easy to avoid, just set `SINGULARITY_CACHEDIR`. The user should know this.
I would just add "This can add a large cache file in your $HOME, to avoid export (or setenv) SINGULARITY_CACHEDIR to the desired one" to the -n, -c, and -v options. I'd be happy to do that, but it's an. unwieldy sentence, it maybe best to group these three and have the sentence just once before them ("The following options...").
Let me know which way, I'll make the PR

**Describe alternatives you've considered**
I mean, one could add another option and set the variable inside the script, but that seems like overkill.

**Additional context**

**Comment by wdconinc** [comment]
I'm quite happy to add some warnings to those running singularity or docker without cvmfs (and provide a link to how to install cvmfs). Those are the 'annoying' modes to support. If we provide better redirect to running singularity against cvmfs I also won't feel as bad about implementing some functionality only for singularity against cvfms (e.g. #2).

So, something like:
```
Notice: singularity stores large files in \$SINGULARITY_CACHEDIR, which is set to:
  $SINGULARITY_CACHEDIR
You are responsible for periodically clearing the contents of this directory.
```
and
```
Notice: You are NOT using cvmfs, and will need to upgrade eic-shell manually.
To use cvmfs, check the Getting Started section at https://cvmfs.readthedocs.io.
```

**Comment by kkauder** [comment]
I agree to the above. One possible addition: The docker installation has a similar problem, not as pronounced because it doesn't happen in farm nodes with small home directories. But users need to regularly `eic-shell --upgrade` AND clean up with `docker image prune`
